tests/test_misc.py

```diff
@@ -155,7 +155,8 @@ def test_unordered_dict(init_func):
     """
     Interceptor configs can only be handled after the plugin has been loaded not before.
     """
-    client = init_func({'confluent.monitoring.interceptor.publishMs': 1000,
+    client = init_func({'group.id': "test-group",
```

Comment:
single quotes

Reply:
Since there doesn't seem to be a strict convention around when to use one or the other I've been trying to come up with what I think feels right. I was thinking that identifiers such as dict keys should be in single quotes while there values should be in double quotes. Thoughts?

Reply:
Right, but there is existing code in this dict that uses single quotes for values, so that is what should be used for consistency.